Update docs to reference notebook sessions (#358)
posit-dev/positron-docs: managing-interpreters.qmd

description: "Manage multiple Python and R interpreter sessions in Positron. Switch between environments, configure startup options, and run multiple sessions simultaneously."
---

Positron is designed to support multiple R and Python interpreter sessions. This enables workflows that span multiple environments, and allows you to switch between them easily within a single workspace.

Positron has two types of interpreter sessions:

* **Console sessions** drive the **Console** pane. Create and manage them from the Interpreter picker or the **Console** pane.
* **Notebook sessions** are tied to a specific notebook. Create and manage them from the notebook editor's Kernel Selector.

> The Interpreter picker lists every running interpreter session so you can see and switch between them from one place.

Reviewer: ```suggestion
The Interpreter picker lists every running interpreter session so you can manage them all from one place.
```

> At any given time, one interpreter session is designated as the **active interpreter session**. This is the language environment currently in use for interactive and language-specific features within the IDE.

Reviewer: ```suggestion
At any given time, one interpreter session is designated as the _active_ interpreter session. This is the language environment currently in use for interactive and language-specific features within the IDE.
```

Probably shouldn't be bold according to the style guide, I think?

Author: Yes I think you are right!
